src/configuration/configuration_controller.py

```diff
@@ -0,0 +1,88 @@
+from plumbum import local, FG, BG, TF, RETCODE, ProcessExecutionError
+import subprocess
+
+
+class ConfiguartionController:
+    def __init__(self):
+        self._multichain_util = local['multichain-util']
+        self._multichain_d = 'multichaind'
+        self._multichain_daemon = "-daemon"
+        self._create = self._multichain_util['create']
+        self._multichain_path = "~/.multichain/"
+        self._params_file = "/params.dat"
```

Comment:
Also, `_multichain_path_` seems to be an absolute path. I expected to see something similar like `_params_path`. @darshgrewal what directories (including sub-directories) does `multichain` check recursively for this `params.dat` file? Right now it looks like a lot like magic to me 😲